presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns position of the N-th instance of substring from the back of a string.
+* Add aggregation function :func:`entropy`.
+* Add classification aggregation functions :func:`classification_miss_rate`, :func:`classification_precision`,
+  :func:`classification_recall`, :func:`classification_thresholds`.
+* Add maximum standard error to :func:`approx_set`.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
```

Comment:
Lets follow the order of changes - https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes . `Fix`s go first